Categories/Category/Monoidal/Braided/Properties.agda

```diff
@@ -0,0 +1,79 @@
+{-# OPTIONS --without-K --safe #-}
+
+open import Categories.Category using (Category)
+open import Categories.Category.Monoidal using (Monoidal)
+open import Categories.Category.Monoidal.Braided using (Braided)
+
+module Categories.Category.Monoidal.Braided.Properties
+  {o ℓ e} {C : Category o ℓ e} {M : Monoidal C} (BM : Braided M) where
+
+open import Data.Product using (_,_)
+
+open import Categories.Category.Monoidal.Properties M
+open import Categories.Functor using (Functor)
+open import Categories.Morphism.Reasoning C
+open import Categories.NaturalTransformation.NaturalIsomorphism.Properties
+  using (push-eq)
+
+open Category C
+open Braided BM
+open Commutation
+open MonoidalReasoning
+
+private
+  variable
+    X Y Z : Obj
+
+  -- A shorthand for the braiding
+  B : ∀ {X Y} → X ⊗₀ Y ⇒ Y ⊗₀ X
+  B {X} {Y} = braiding.⇒.η (X , Y)
+
+-- sstucki: what is the intuition behind this lemma?  Why introduce it
```

Comment:
it's used once, it's probably OK to get pushed inside of `braiding-coherence⊗unit` but I don't find it's an essential problem. I do think it's a good idea to hide these two helper lemmas though, so we just expose `braiding-coherence`. just some indentation from line 28 to 65 would work.

Reply:
I changed the comment and made the helpers private.